### PR TITLE
[Broadcaster] Add chat moderation capabilities

### PR DESCRIPTION
**Pull request by LVala**
It also renames `/admin/player` to `/admin/panel`.

### CODE REVIEW

File: broadcaster/lib/broadcaster_web/channels/stream_channel.ex [hunk 1]
```diff
@@ -11,22 +11,39 @@ defmodule BroadcasterWeb.StreamChannel do
...
 
   @impl true
   def handle_in("join_chat", %{"nickname" => nickname}, socket) do
     case register(nickname) do
       :ok ->
-        socket = assign(socket, :nickname, nickname)
+        socket =
+          socket
+          |> assign(:nickname, nickname)
+          |> assign(:id, 0)
```
Comment (mickel8) [inline broadcaster/lib/broadcaster_web/channels/stream_channel.ex:45]: Better start with a random value

File: broadcaster/lib/broadcaster_web/channels/stream_channel.ex [hunk 2]
```diff
@@ -11,22 +11,39 @@ defmodule BroadcasterWeb.StreamChannel do
     {:ok, assign(socket, :nickname, nil)}
   end
 
+  @impl true
+  def join("stream:chat-admin", _, socket) do
+    {:ok, socket}
+  end
```
Comment (mickel8) [inline broadcaster/lib/broadcaster_web/channels/stream_channel.ex:17]: Let's go for counting streamer as viewer and remove this

**Review by mickel8 — APPROVED** [review]

File: broadcaster/lib/broadcaster_web/channels/stream_channel.ex (hunk 1 above)
Comment (LVala) [reply broadcaster/lib/broadcaster_web/channels/stream_channel.ex:45]: Instead, I'll use completely unique id to create the message id.

Comment (mickel8) [reply broadcaster/lib/broadcaster_web/channels/stream_channel.ex:39]: Nitpick:

```suggestion
          |> assign(:msg_cnt, 0)
```

File: broadcaster/lib/broadcaster_web/router.ex [hunk 3]
```diff
@@ -44,6 +44,12 @@ defmodule BroadcasterWeb.Router do
       post "/sse", MediaController, :sse
       post "/layer", MediaController, :layer
     end
+
+    scope "/admin" do
+      pipe_through :auth
+
+      delete "/chat/:id", PageController, :delete_chat_message
+    end
```
Comment (mickel8) [inline broadcaster/lib/broadcaster_web/router.ex:52]: I think we can merge this with already exisiting scoep "admin", can't we?

**Review by mickel8 — APPROVED** [review]

File: broadcaster/lib/broadcaster_web/router.ex (hunk 3 above)
Comment (LVala) [reply broadcaster/lib/broadcaster_web/router.ex:52]: The `admin` scope is piped through `:browser`, so we cannot use it for that. I could refactor the `admin` scope a bit, but I'm not sure if it's worth it.

**Pull request merged**